src/outcomeFactory.js

```diff
@@ -0,0 +1,41 @@
+const FinalOutcome = require('./finalOutcome')
+const ActionableOutcome = require('./actionableOutcome')
+const NullOutcome = require('./nullOutcome')
+const MarkerFinder = require('./markerFinder')
+
+module.exports = class OutcomeFactory {
+  constructor(board) {
+    this.finder = new MarkerFinder(board)
```

Comment:
Also, I still don't think I'm grasping how the dependency structure should look for the board. The board's interface provides methods so that there aren't any other modules digging through the array, but I think there are still too many classes, both low level and high, that require the board.